RELEASE/scripts/autoscend.ash

```diff
@@ -1979,6 +1979,10 @@ boolean doTasks()
 	
 	if (process_tasks()) return true;
 
+	if(meatreserve() > 0)
```

Comment:
trivial perf improvement: calculate this once, store it, then use the calculated version. It won't change between comparing to 0 and logging.